packages/common/src/types/index.ts

```diff
@@ -610,7 +619,7 @@ export type SandboxUrlSourceData = {
   git?: GitInfo;
 };
 
-export type LiveMessage<data = undefined> = {
+export type LiveMessage<data = any> = {
```

Comment:
Use `unknown` instead to have better type-safety

```suggestion
export type LiveMessage<data = unknown> = {
```